src/streaming/rules/scheduling/BufferLevelRule.js

```diff
@@ -31,12 +31,13 @@
 import MediaPlayerModel from '../../models/MediaPlayerModel';
 import PlaybackController from '../../controllers/PlaybackController';
 import FactoryMaker from '../../../core/FactoryMaker';
+//import Debug from '../../../core/Debug';
 
 function BufferLevelRule(config) {
 
     let instance;
     let context = this.context;
-
+    //let log = Debug(context).getInstance().log;
```

Comment:
I can clean that up. I did mean to keep it in as I enable disable logs in the object a lot and was tired of adding code and import over and over just to remove before commit.  But dirty so Ill clean.